meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend

```diff
@@ -13,13 +13,13 @@
 #
 # Enable CAN bus network by default
 #
-FILESEXTRAPATHS:prepend := "${THISDIR}/systemd-conf-files:"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
 SRC_URI += " file://can0.network"
 
 FILES:${PN} += " ${sysconfdir}/systemd/network/can0.network"
 
 do_install:append() {
-    install -d ${D}${sysconfdir}/systemd/network
-    install -m 0644 ${WORKDIR}/can0.network ${D}${sysconfdir}/systemd/network
-}
\ No newline at end of file
+    install -D -m 0644 ${WORKDIR}/can0.network ${D}${systemd_unitdir}/network
+}
```

Comment:
This mean for now that we will not have CAN

Reply:
I checked the parent recipe and saw that our implementation was using the incorrect paths on the device (the conf folder of systemd)

The default can0 network systemd network unit file is supposed to be in /lib/systemd/network, which is ${systemd_unitdir}